client/main.lua

```diff
@@ -226,14 +226,24 @@ end
 ---@param args table<string, any>
 local function openVehicleCategoryMenu(args)
     local categoryMenu = {}
+    local sortedCategories = {}
     for k, v in pairs(config.shops[insideShop].categories) do
-        categoryMenu[#categoryMenu + 1] = {
-            title = v,
-            arrow = true,
-            onSelect = function()
-                openVehCatsMenu(k, args.targetVehicle)
+        table.insert(sortedCategories, v)
```

Comment:
```suggestion
        sortedCategories[#sortedCategories + 1] =  v
```

Reply:
Didnt mean to close it. My bad Im used to glua

Reply:
When changing line 231 to your suggested, It breaks the script. 

MainThrd/ ^1SCRIPT ERROR: @qbx_vehicleshop/client/main.lua:458: attempt to index a nil value (field '?')^7

Reply:
That sounds strange. Line 458 doesn't seem to be related to sortedCategories. Do you have a different line 458?